firmware/boron-ins-fsm/src/im21door.cpp

```diff
@@ -174,6 +174,25 @@ void threadBLEScanner(void *param) {
         scanThreadDoorData.doorStatus = doorAdvertisingData[5];
         scanThreadDoorData.controlByte = doorAdvertisingData[6];
 
+        // Door heartbeat message: 
+        // dooradvertisingdata[0] (1 byte): Firmware Version
+        // dooradvertisingdata[1-3] (3 bytes): Last 3 bytes of door sensor address
+        // dooradvertisingdata[4] (1 byte): Type ID (door sensor, thermometer, smoke detector, etc.)
+        // dooradvertisingdata[5] (1 byte): Event Data
+          // bit[3]: heartbeat?
+          // bit[2]: low battery?
+          // bit[1]: door open?
+          // bit[0]: tamper alarm?
+        // dooradvertisingdata[6] (1 byte): Control Data
+        // More info: https://drive.google.com/file/d/1ZbnHi7uA_xMWVIiMbQjZlbT3OOoykbr4/view?usp=sharing        
+
+        if (stateMachineDebugFlag) { //(doorAdvertisingData[5] & (1 << 3)) != 0 &&
```

Comment:
Should this be uncommented so that the debug message is only sent after a heartbeat message? `//(doorAdvertisingData[5] & (1 << 3)) != 0 &&` 

Is this going to send three times if the message arrives on all three channels?

Reply:
Yes I should make the change to that it only sends for heartbeat messages. 
The publishes will happen 1-4 for each of the channels, I can avoid that by moving it so that it is processed in `checkIM21`, which bypasses the os_queue. However at that point, it's pretty much the same data as the door data seen in the normal debug messages.

Reply:
No, you're right, nicer to have it here. And since we aren't going to always be in debug mode, maybe it makes sense to have it published literally every time. I think I'm OK with it as it is.

Reply:
By "every time", do you mean every door heartbeat or every door broadcast?

Reply:
Haha, ambiguous, I'm not even sure which of those two is the one that I want.

What I mean is to not do anything fancy to deduplicate messages that we get more than once on different channels.

Reply:
I see. I'll go with the former then.